charts/consul/templates/gateway-resources-job.yaml

```diff
@@ -99,6 +99,7 @@ spec:
             - {{- toYaml .Values.connectInject.apiGateway.managedGatewayClass.copyAnnotations.service.annotations | nindent 14 -}}
             {{- end }}
             - -service-type={{ .Values.connectInject.apiGateway.managedGatewayClass.serviceType }}
+            - -openshift-scc-name={{ .Values.connectInject.apiGateway.managedGatewayClass.openshiftSCCName }}
```

Comment:
I think it makes sense to gate this to only if `global.openshift.enabled = true`